chart/templates/operator.yaml

```diff
@@ -242,27 +260,11 @@ spec:
             - group: addon.open-cluster-management.io
               resource: addondeploymentconfigs
           EOF
-      containers:
-      - args:
-        - controller
-        env:
-        - name: STATUS_ADDDON_IMAGE_NAME
-          value: ko.local/ocm-status-addon:to_be_replaced
-        image: ko.local/ocm-status-addon:to_be_replaced
-        imagePullPolicy: IfNotPresent
-        name: status-controller
-        resources:
-          limits:
-            cpu: 500m
-            memory: 512Mi
-          requests:
-            cpu: 10m
-            memory: 64Mi
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-            - ALL
+        command:
+        - sh
+        - -c
+        image: quay.io/kubestellar/kubectl:1.29.3
```

Comment:
please upgrade to 1.30.1
https://quay.io/repository/kubestellar/kubectl?tab=tags

Reply:
I would like to get everything to 1.30, but I think we have to do it in a few steps. For one thing, the factoring into multiple repos forces sequencing along the lines of dependencies. For another, IIRC it is not recommended to have kubectl more than one minor release higher than the apiserver. So I think the ambitions for the first step here have to be limited.

Reply:
To clarify, this version of kubectl does not have anything to do with the customer, it only operates in the cluster. It's being used by the core chart already without any problem. Upgrading the version reduces the number of images to pull

Reply:
Yes, and there are version skew constraints between kubectl and apiserver, and version skew constraints between apiserver and clients (i.e., our controllers).

Can we pursue this as a separate issue?

Reply:
I agree with @MikeSpreitzer, since there could be version skew issues with the API servers we use let's address this in a separate issue.